Source/System Explorer/Embedded Data Logger System Explorer.lvlib

```diff
@@ -1,44 +1,48 @@
-﻿<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
LabVIEW decided to put spaces at the end of all of the lines, causing the diff to be the entire file. Diff without whitespace for a more meaningful diff.